gnc/sub8_thruster_mapper/nodes/mapper.py

```diff
@@ -82,6 +85,10 @@ def get_thruster_wrench(self, position, direction):
         wrench_column = np.hstack([forces, torques])
         return np.transpose(wrench_column)
 
+    def get_b_matrix(self, srv):
```

Comment:
We should probably thread_lock this lest we accidentally send B while it is being generated. 

That, or only start advertising the service once we've generated B for the first time